xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp

```diff
@@ -98,6 +98,18 @@ void CGUIDialogVideoManagerVersions::UpdateButtons()
   }
 }
 
+void CGUIDialogVideoManagerVersions::UpdateDefaultViedeoVersionSelection()
+{
+  // find new item in list and select it
+  for (const auto& item : *m_videoAssetsList)
+  {
+    if (item->GetVideoInfoTag()->m_iDbId == m_defaultVideoVersion->GetVideoInfoTag()->m_iDbId)
```

Comment:
You can probably assign this to a variable and pass it to select. Or use the condition directly in select.
Up to you, readibility might decrease but you kill the if/else branches

Reply:
Yep, will change that.